### PR TITLE
Add initial django files and configure them for passing tests

### PR DESCRIPTION
**Pull request by mikescor**
Add django files, change max-line-length argument to 120 for passing flask8 test, add docstrings to django files for passing pep257 test.

### CODE REVIEW

**Comment by Miseracle** [comment]
Please, fill the description and PR title with valid info about PR.

File: blog/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+"""Don't write here."""
```
Comment (webknjaz) [inline blog/__init__.py]: It's a stupid docstring. Why would you even need this module?

File: blog/apps.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+"""Configure app here."""
```
Comment (webknjaz) [inline blog/apps.py]: The docstring must describe the module and not be some call to action.

File: blog/apps.py [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+from django.apps import AppConfig
+
+
+class BlogConfig(AppConfig):
+    """Configuration."""
```
Comment (webknjaz) [inline blog/apps.py]: Be more specific.

File: blog/forms.py [hunk 4]
```diff
@@ -0,0 +1,12 @@
+"""This module keeps the collection of form declarations representing HTML forms."""
```
Comment (webknjaz) [inline blog/forms.py]: This is too verbose.

File: blog/forms.py [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+
+from django import forms
+from .models import Post
+
+
+class PostForm(forms.ModelForm):
+    """Form for Post model."""
```
Comment (webknjaz) [inline blog/forms.py]: No. It's a form for Post entity. Don't write obvious things I can see two lines below.

File: blog/migrations/0001_initial.py [hunk 6]
```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.6 on 2017-03-02 20:41
+"""Migrations."""
```
Comment (webknjaz) [inline blog/migrations/0001_initial.py]: Normally, autogenerated things aren't being checked by linter.

Comment (webknjaz) [reply blog/migrations/0001_initial.py]: And it's a single migration, not multiple.

File: blog/migrations/__init__.py [hunk 7]
```diff
@@ -0,0 +1 @@
+"""Migrations."""
```
Comment (webknjaz) [inline blog/migrations/__init__.py]: O'rly?

File: blog/models.py [hunk 8]
```diff
@@ -0,0 +1,22 @@
+"""Models for project."""
+from django.db import models
+from django.utils import timezone
+
+
+class Post(models.Model):
+    """Model for post in blog."""
```
Comment (webknjaz) [inline blog/models.py]: It's obvious that it's a model. I can see it a line above.

File: blog/models.py [hunk 9]
```diff
@@ -0,0 +1,22 @@
...
+    created_date = models.DateTimeField(default=timezone.now)
+    published_date = models.DateTimeField(blank=True, null=True)
+
+    def publish(self):
+        """Publish the post."""
+        self.published_date = timezone.now()
+        self.save()
+
+    def __str__(self):
+        """Return title of the post."""
```
Comment (webknjaz) [inline blog/models.py]: Really? Isn't it what is stated in the next line?

File: blog/views.py [hunk 10]
```diff
@@ -0,0 +1,48 @@
+"""All views of project."""
```
Comment (webknjaz) [inline blog/views.py]: I think this description is meaningless.

File: blog/views.py [hunk 11]
```diff
@@ -0,0 +1,48 @@
...
+from django.shortcuts import render, get_object_or_404, redirect
+from django.utils import timezone
```
Comment (webknjaz) [inline blog/views.py]: It's nice to separate import sections with an empty line here.

File: blog/views.py [hunk 12]
```diff
@@ -0,0 +1,48 @@
...
+
+
+def post_detail(request, pk):
+    """Show details about post."""
+    post = get_object_or_404(Post, pk=pk)
+    return render(request, 'blog/post_detail.html', {'post': post})
+
+
+def post_new(request):
+    """Return page with form for adding new post."""
```
Comment (webknjaz) [inline blog/views.py]: I think it's "render". Well, maybe "show".

File: blog/views.py [hunk 13]
```diff
@@ -0,0 +1,48 @@
...
+            post.published_date = timezone.now()
+            post.save()
+            return redirect('blog.views.post_detail', pk=post.pk)
+    else:
+        form = PostForm()
+    return render(request, 'blog/post_edit.html', {'form': form})
+
+
+def post_edit(request, pk):
+    """Return page for editing post."""
```
Comment (webknjaz) [inline blog/views.py]: "return" looks ugly here as well.

File: manage.py [hunk 14]
```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
+"""Manage settings of project."""
```
Comment (webknjaz) [inline manage.py]: Oh no, it manages the app, not its settings.

File: mysite/__init__.py [hunk 15]
```diff
@@ -0,0 +1 @@
+"""Don't write here."""
```
Comment (webknjaz) [inline mysite/__init__.py]: Really? :hankey: :hankey: :hankey:

File: mysite/urls.py [hunk 16]
```diff
@@ -0,0 +1,11 @@
+"""Configuration of mysite project URLs."""
+
+from django.conf.urls import url
+from . import views
+
+urlpatterns = [
```
Comment (webknjaz) [inline mysite/urls.py]: You must import urls from blog rather than duplicating them.

**Review by webknjaz — CHANGES_REQUESTED** [review]

File: blog/apps.py [hunk 17]
```diff
@@ -1,8 +1,8 @@
-"""Configure app here."""
+"""Configuring application."""
```
Comment (webknjaz) [inline blog/apps.py]: It it a process or a module?

Comment (webknjaz) [inline blog/apps.py]: It it a process or a module?

File: blog/migrations/0001_initial.py [hunk 18]
```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 # Generated by Django 1.10.6 on 2017-03-02 20:41
-"""Migrations."""
+"""Migration settings."""
```
Comment (webknjaz) [inline blog/migrations/0001_initial.py]: no.

File: blog/models.py [hunk 19]
```diff
@@ -18,5 +18,5 @@ def publish(self):
         self.save()
 
     def __str__(self):
-        """Return title of the post."""
+        """Get title of the post."""
```
Comment (webknjaz) [inline blog/models.py]: No.

File: blog/views.py [hunk 20]
```diff
@@ -1,4 +1,5 @@
-"""All views of project."""
+"""Configuration for views."""
```
Comment (webknjaz) [inline blog/views.py]: no.

File: blog/views.py [hunk 21]
```diff
@@ -18,7 +19,7 @@ def post_detail(request, pk):
 
 
 def post_new(request):
-    """Return page with form for adding new post."""
+    """Get page with form for adding new post."""
```
Comment (webknjaz) [inline blog/views.py]: no

File: blog/views.py [hunk 22]
```diff
@@ -33,7 +34,7 @@ def post_new(request):
 
 
 def post_edit(request, pk):
-    """Return page for editing post."""
+    """Get page for editing post."""
```
Comment (webknjaz) [inline blog/views.py]: no.

File: manage.py [hunk 23]
```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env python
-"""Manage settings of project."""
+"""App management."""
```
Comment (webknjaz) [inline manage.py]: it's not a process. see pep257

Comment (webknjaz) [inline manage.py]: it's not a process. see pep257

**Review by webknjaz — CHANGES_REQUESTED** [review]
* Rethink the purpose of docstrings, read PEP 257;
* Fix current comments;
* Fix previous comments as well;
* Write sane commit messages and docstrings.

File: blog/migrations/0001_initial.py [hunk 24]
```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.6 on 2017-03-02 20:41
+"""Initial migration. Autocreated.
```
Comment (DrEdi) [inline blog/migrations/0001_initial.py:3]: you don't have to write this for autocreated files

File: blog/models.py [hunk 25]
```diff
@@ -18,5 +18,5 @@ def publish(self):
         self.save()
 
     def __str__(self):
-        """Get title of the post."""
+        """Convert an object to a string."""
```
Comment (webknjaz) [inline blog/models.py]: "Render Post as its title."

**Review by webknjaz — CHANGES_REQUESTED** [review]

File: .pre-commit-config.yaml [hunk 26]
```diff
@@ -10,6 +10,7 @@
     -   id: double-quote-string-fixer
     -   id: end-of-file-fixer
     -   id: flake8
+        exclude: migrations
```
Comment (webknjaz) [inline .pre-commit-config.yaml]: you'd better use full relative path `blog/migrations` to prevent accidentally ignoring smth important.

**Review by webknjaz — CHANGES_REQUESTED** [review]

**Review by webknjaz — APPROVED** [review]
LGTM

**Pull request merged**